addons/mail/static/src/components/composer_suggested_partners/composer_suggested_partners.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.ComposerSuggestedPartners" owl="1">
+        <!-- List of followers -->
+        <div class="o_composer_suggested_partners">
+            <t t-if="state.more" t-foreach="thread.suggestedPartners" t-as="recipient" t-key="partner_id">
```

Comment:
- no (pseudo) tag combining `t-if` and `t-foreach`. Make 2 distinct (pseudo) tags instead
- `t-key` should be unique using the `t-as` variable, so should be `recipient.partner_id`.